android/java/org/chromium/chrome/browser/brave_news/models/FeedItemCard.java

```diff
@@ -0,0 +1,50 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.chromium.chrome.browser.brave_news.models;
+
+import org.chromium.brave_news.mojom.CardType;
```

Comment:
`import org.chromium.brave_news.mojom.CardType;` is not used, here I see it replaced with `int`

Reply:
removed